### DISCUSSION TITLE
New to GitHub.com / non-paid account

### DISCUSSION THREAD

**Discussion by dstin44** [community/community#22902] [New to GitHub]
<p>"Don’t need unlimited private repositories? <a href="https://github.com/join?source=pricing-page-new" rel="nofollow noopener">Sign up for one of our free plans</a> for unlimited public and open source projects. "</p>
<p>I got this account and I want to learn the <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> for PHP work. Will this be possible, meaning will I have the access to all the functionalites that I would be able to use with the paid one (or like the most of them). I mean, I know that the number of repositories will be limited, but will I be able to learn like 100% of the site (or how much percent).</p>
<p>Also, how would I learn PHP coding from this site? How would I look for other people’s code, how to browse that, to benefit educationally as far as the PHP (I am like intermediete level now). I mean, what is the “culture” like here of this, where do I go, how do I find what I need to find. Like the mindset is code in an IDE (I have phpStorm), publish to <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a>, look at other’s people code and so on. Make the code on my account better and better over time. Basically the learning, but using the “spirit of this site” (how) for that. </p>
<p>Thanks. </p>

**Comment by mpboom** [comment, marked as answer]
<p><strong>Free / paid accounts</strong></p>
<p>You’ll <em>always</em> have access to _all _features and can create <em>unlimited</em> repositories. The <em>only</em> limitation for free accounts is that you can’t have <em>private</em> repositories in your account.</p>
<p><strong>Learning languages</strong></p>
<p>Well, I am not sure if GitHub is the best place to really start learning a language, but since you already know the language you could just try to <em>contribute</em> to existing PHP projects. This way you’ll both help out the community, and also learn. As a free bonus you’ll probably receive feedback on your code. You can browse PHP projects at the PHP <em>topic</em>: <a href="https://github.com/topics/php" rel="nofollow noopener">https://github.com/topics/php</a> .</p>
